src/components/views/rooms/RoomSublist.tsx

```diff
@@ -917,3 +919,4 @@ export default class RoomSublist extends React.Component<IProps, IState> {
         );
     }
 }
+45
```

Comment:
Please undo this new line to minimise changes to the code and minimise merge conflicts with the default element code.

Reply:
done

Reply:
The line above now has a change, please revert the change to that line too.

Reply:
Hi Jasen, I am not sure why it's showing it as a change even though there is no change. I tried out pulling the master branch to see in case I get any merge issues. but I am not getting any merge issues. 
Maybe you need to guide me a bit on why it's displaying it as a change on line 921